### PR TITLE
Several upgrades

### PR DESCRIPTION
**Pull request by alix-tz**
Linked to issues #87, #92 and #93

- added a Models page
- added Publications in Team as well as ORCID ids for both of us
- finished minor fix in "authors" blocks

### CODE REVIEW

**Comment by alix-tz** [comment]
squash before merge

File: assets/vanilla-i18n/htr-united_i18n.csv [hunk 1]
```diff
@@ -120,4 +121,8 @@ actions.step2,(2) Add this file in your repository in the folder ,(2) Ajoutez le
 actions.add,Add the file on ,Ajouter la configuration sur
 actions.edit,Edit the file if it exists ,Éditer la configuration si elle existe déjà sur 
 actions.addBadge,(3) Add the following text to your README.md file if you want to show your badges,(3) Ajoutez le texte suivant à votre fichier README si vous voulez afficher les badges
-actions.editReadme,Edit the README.MD on Github,Éditez le Readme.md sur Github
\ No newline at end of file
+actions.editReadme,Edit the README.MD on Github,Éditez le Readme.md sur Github
+models.title,Models,Modèles
+models.p1,"The HTR-United catalog does not directy list transcription modèles, but rather the data that were used to create such models. You can explore the repositories linked to the datasets in which you are interested in order to find out if they link to a public model.","Le catalogue d'HTR-United ne référence pas directement de modèles de transcription, mais plutôt les données ayant permis d'en produire. Vous pouvez consulter les répertoires des jeux de données qui vous intéressent pour savoir s'ils renvoient vers un modèle public."
```
Comment (PonteIneptique) [inline assets/vanilla-i18n/htr-united_i18n.csv:126]: I think you meant "models"  ;)

File: team.html [hunk 2]
```diff
@@ -74,30 +86,38 @@
...
 	  		</div>
 	  		<div class="col-md-2">
 	  			<img src="https://pbs.twimg.com/profile_images/922043022916902913/Xdvaypzu_400x400.jpg" class="img-fluid rounded" alt="Photo de Thibault Clérice" />
 	  		</div>
 	  	</section>
+
+      <h1 class="mt-4 mb-3">Publications</h1>
+      <ul class="list-group">
```
Comment (PonteIneptique) [inline team.html:112]: Le texte n'est pas visible chez moi à cause du footer ? 
image

File: assets/vanilla-i18n/compile.py [hunk 3]
```diff
@@ -56,5 +56,5 @@ def treat_csv(filename="htr-united_i18n.csv", dictionary=None):
 
 # Dump output
 for lang in Output:
-	with open(f"{lang}.json", "w") as f:
+	with open(f"{lang}.json", "w", encoding="utf8") as f:
```
Comment (PonteIneptique) [inline assets/vanilla-i18n/compile.py:59]: Hehe

**Review by PonteIneptique — CHANGES_REQUESTED** [review]
Quelques petits changements sinon génial ;)

File: assets/vanilla-i18n/compile.py (hunk 3 above)
Comment (alix-tz) [reply assets/vanilla-i18n/compile.py:59]: A real nightmare on windows...

File: team.html (hunk 2 above)
Comment (alix-tz) [reply team.html:112]: Then I'll remove the sticky footer on that page

Comment (alix-tz) [reply team.html:112]: I mean I just want it to stay at the bottom of the page in stead of being in the middle when the page is too short...

**Comment by alix-tz** [comment]
Should be all good now, can I let you merge?

**Pull request merged**